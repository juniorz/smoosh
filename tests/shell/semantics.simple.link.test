set -e
echo 'echo hi' >cmd.sh
chmod +x cmd.sh
ln -s cmd.sh link.sh
OLDPATH=$PATH
PATH=.
[ -x cmd.sh ]
[ -L link.sh ]
cmd.sh  # command works
link.sh # symlink works
PATH=$OLDPATH
ls
rm cmd.sh link.sh