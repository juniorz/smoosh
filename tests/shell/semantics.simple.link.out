hi
hi
cmd.sh
link.sh
